### PR TITLE
Command dedup conformance suites readme

### PR DESCRIPTION
**Pull request by nicu-da**
Add readme which explains the differences between different command duplication suites

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/ledger-api-test-tool/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Suites
+
```
Comment (fabiotudone-da) [inline ledger/ledger-api-test-tool/README.md:2]: I'd consider removing this at this stage

File: ledger/ledger-api-test-tool/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+### Command deduplication suites
```
Comment (fabiotudone-da) [inline ledger/ledger-api-test-tool/README.md:3]:
```suggestion
### Command deduplication test suites
```

Can you add a short intro to give context, e.g. why there are different suites for command deduplication in the first place?

File: ledger/ledger-api-test-tool/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+* append-only - Requires the schema to be append-only because we use the submission id set in the completion, which is
+  present only for append-only schemas
+* configuration-required - If it requires specific settings to be set for the ledger configuration
+
+| Name | Default | Append only | Configuration required | Details |
+| --- | --- | --- | --- | --- |
+|CommandDeduplicationIT|Yes|No| No  |Tests participant deduplication|
+|KVCommandDeduplicationIT| No | No | minSkew set to 1 second. maxDeduplicationDuration has to be < 5s | Extends the test cases from `CommandDeduplicationIT` with committer side test cases. Requires the time model update because KV committer deduplication is based on maxDeduplicationDuration + minSkew|
+|AppendOnlyKVCommandDeduplicationIT|No|Yes|Same as KVCommandDeduplicationIT | Same as `KVCommandDeduplicationIT` but it requires an append-only schema so that we have access to the submission id|
+|AppendOnlyCommandDeduplicationParallelIT | No |Yes| trackerRetentionPeriod has to be set to <= 5s | Requires append only schema so that we have access to the submission id |
```
Comment (fabiotudone-da) [inline ledger/ledger-api-test-tool/README.md:15]: Why is such a configuration needed?

Comment (nicu-da) [reply ledger/ledger-api-test-tool/README.md:15]: Good point, that was required as a fix for DPP-609, but that's done now so we can actually remove those constrains

Comment (nicu-da) [reply ledger/ledger-api-test-tool/README.md:15]: Will open a PR which cleans the usage of that setting

Comment (nicu-da) [reply ledger/ledger-api-test-tool/README.md:15]: https://github.com/digital-asset/daml/pull/11061

**Review by fabiotudone-da — APPROVED** [review]

**Pull request merged**